deployment/document-understanding-solution.template

```diff
@@ -655,7 +659,7 @@ Resources:
                     },
                     {
                         "Effect": "Allow",
-                        "Resource": "arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:*",
+                        "Resource": "*",
```

Comment:
Just curious, can we switch this to : 
arn:aws:cloudformation:*:${AWS::AccountId}:* ?